R/keyword_search.r

```diff
@@ -76,26 +78,46 @@ keyword_search <- function(x, keyword, path = FALSE, split_pdf = FALSE,
                            remove_hyphen = TRUE, token_results = TRUE,
                            heading_search = FALSE, heading_args = NULL,
                            convert_sentence = TRUE, 
-                           remove_equations = TRUE,
-                           split_pattern = "\\p{WHITE_SPACE}{3,}", ...) {
+                           remove_equations = FALSE,
+                           split_pattern = "\\p{WHITE_SPACE}{3,}",
+                           ocr_pdf_fun = function(...) {return("")},
+                           ocr_pdf_fun_params = nil,
+                           translate_fun = function(x,...) {return(x)},
+                           translate_fun_params = nil,
+                           translation_target_language="eng",
+                           ...) {
+  print(x)

```

Comment:
@behrica was this a debugging step? Any reason why x would need to be printed here for the OCR to happen?